submissions/cynthiaNwakaeme/php/loopAndNums.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+    $threshold = 101;
+
+    echo(" A PHP program that displays every number between 100 and 150 <br>");
+    
+        for ($i = $threshold; $i < 150; ++$i) {
+            echo $i.'<br>';
+        };
+
+    echo ("<br>A PHP program that displays all even number between 0 and 50 <br>");
+        
+        for ($i = 1 ; $i < (($threshold-1)/2); ++$i){
```

Comment:
start your count from `2` instead of `1`

Reply:
Hello ma @adaugochi the output starts from 2, does that still require modification?